app/forms/HasSecondaryContactFormProvider.scala

```diff
@@ -25,6 +25,6 @@ class HasSecondaryContactFormProvider @Inject() extends Mappings {
 
   def apply(primaryContactName: String): Form[Boolean] =
```

Comment:
param primaryContactName is no longer used and should be removed, and tests updated.